Core/src/org/sleuthkit/autopsy/keywordsearchservice/KeywordSearchService.java

```diff
@@ -48,13 +48,19 @@
      * all of its attributes.
      *
      * @param artifact The artifact to index.
+     * 
+     * @deprecated postArtifact org.sleuthkit.autopsy.datamodel.Blackboard will
+     * take care of this functionality at ingest
      *
      * @throws org.sleuthkit.datamodel.TskCoreException
      */
+    @Deprecated 
     public void indexArtifact(BlackboardArtifact artifact) throws TskCoreException;
 
     /**
-     * Add the given Content object to the text index.
+     * Add the given Content object to the text index.  This message should only
+     * be used in atypical cases, such as indexing a report.  Artifacts are indexed 
+     * when postArtifact called and files are indexed during ingest.
```

Comment:
Missing an "is" between "postArtifact" and "called." I would also like to see postArtifact qualified with the class name. Perhaps "org.sleuthkit.autopsy.datamodel.Blackboard.postArtifact."

Reply:
I would still like to see postArtifact qualified here.